e2e/aks-runtime/aks_runtime_test.go

```diff
@@ -39,7 +39,7 @@ func TestAKSRuntime(t *testing.T) {
 	kataPolicyGenV, err := az.KataPolicyGenVersion()
 	require.NoError(err)
 	rg := os.Getenv("azure_resource_group")
-	nodeImageV, err := az.NodeImageVersion(rg, rg)
+	nodeImageV, err := az.NodeImageVersion(context.Background(), rg, rg)
```

Comment:
Since https://github.com/edgelesssys/contrast/pull/1421 we can use `t.Context()`.